packages/modules/src/modules/epics/ContributionsEpic.tsx

```diff
@@ -141,23 +143,38 @@ interface EpicHeaderProps {
     text: string;
     numArticles: number;
     tracking?: OphanTracking;
+    showAboveArticleCount: boolean;
 }
 
 const EpicHeader: React.FC<EpicHeaderProps> = ({
     text,
     numArticles,
     tracking,
+    showAboveArticleCount,
 }: EpicHeaderProps) => {
-    const elements = replaceArticleCount(text, numArticles, 'epic', tracking);
+    const elements = replaceArticleCount(
+        text,
+        numArticles,
+        'epic',
+        tracking,
+        !showAboveArticleCount,
+    );
```

Comment:
I guess this slightly changes the behaviour of existing epics configured in the tool that have both ac above & use the ac placeholder in the copy - do we think that's an issue?

Reply:
Correct. I'm not sure we ever use both at the same time (until now!), but it seems reasonable to always apply this behaviour rather than have 2 opt-out UIs